### ISSUE TITLE
Task: Revisit the recommended articles, courses, and videos

### ISSUE THREAD

**Issue by alshakero** [Automattic/wp-calypso#68582]
### Details

When the Help Center was conceived, the educational material was different from today. We need to revisit what we show in the Help Center.

**Pieces of info:**

1. **Recommended resources**: this lists a few articles based on the Calypso section. The full list and the curation logic lives [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/data-stores/src/contextual-help/contextual-help.tsx). For starters, we should show the article's title instead of hardcoding it.
2. **Videos**: the list is hard-coded [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/components/help-center-more-resources.tsx). 
3. **Courses**: the list is hard-coded [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/components/help-center-more-resources.tsx). 
4. **Step-by-step guides**: should point to https://wordpress.com/support/ instead of https://wordpress.com/learn/.

**What should we do?**
We appreciate any guidance on what's should we show there instead. 

cc @donalirl

**Comment by donalirl** [comment]
>Recommended resources: this lists a few articles based on the Calypso section. The full list and the curation logic lives [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/data-stores/src/contextual-help/contextual-help.tsx). For starters, we should show the article's title instead of hardcoding it.

I agree it would be helpful to show the article's title, because the content of the docs can change like in the What are blocks? example - ref: p7DVsv-fpN-p2#comment-42269. We regularly rewrite the titles of docs to be more actionable.

I scanned the list. Based on what I saw, the suggested docs are _mostly_ good. However, there are some instances where we have more relevant suggested articles to show, like in the case of "Offline Editing" which shows when a user is in the editor - ref: p7DVsv-fpN-p2#comment-42270. In addition, as WordPress.com continues to evolve, the suggested articles we're showing now may not be the best articles to show in 6 or 12 months from now.

Would it be possible to make changes to the suggested articles here @alshakero? If so, is there an easy way to programmatically put this list into a more readable format, like a spreadsheet? For example:

| Calypso | Link |
| --- | --- |
| media | https://wordpress.com/support/media/ |
|   | https://wordpress.com/support/finding-free-images-and-other-media |
|   | https://wordpress.com/support/wordpress-editor/blocks/image-block |

This would make it easier for us to read and identify which article links we would want to swap. Let me know what approach would work best for you!

> 4\. **Step-by-step guides**: should point to https://wordpress.com/support/ instead of https://wordpress.com/learn/.

Yes, this makes sense 👍

**Comment by rickmgithub** [comment]
One thing to note is that the WPCOURSES.com site is now going to be shut down and the content moved across to WordPress.com/learn. It's note clear how that will be structured but once we know more we can update this thread.

**Comment by escapemanuele** [comment]
I think this may not be actual anymore
